src/ameba/source.cr

```diff
@@ -57,6 +57,24 @@ module Ameba
     # source.ast
     # ```
     getter ast : Crystal::ASTNode do
+      code = @code
+
+      {% if compare_versions(Crystal::VERSION, "1.15.0") >= 0 %}
```

Comment:
since this is copied in multiple places, what do you thing about following more declarative way and defining the helper method. Something like:

```crystal
def ecr_supported?
  {{ compare_versions(Crystal::VERSION, "1.15.0") >= 0 }}
end
```